tests/helpers/boring_model.py

```diff
@@ -100,7 +100,7 @@ def step(self, x):
         out = torch.nn.functional.mse_loss(x, torch.ones_like(x))
         return out
 
-    def training_step(self, batch, batch_idx):
+    def training_step(self, batch, batch_idx, dataloader_idx=0):
```

Comment:
Pretty sure this should not be added here. 99% of time we won't have multiple dataloaders. There may be specific tests that can add this in an override.